### PR TITLE
✨ Source MongoDB V2: Clean fieldPath before use in projection

### PR DESCRIPTION
**Pull request by jdpgrailsdev**
## What
* Fix issue with invalid FieldPath synatx preventing syncs from executing

## How
* Remove preceeding "$" from field name before building projection clause.

## Recommended reading order
1. `MongoUtils.java`

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  destination-csv test report (commit [`58e1ec7ad3`](https://github.com/airbytehq/airbyte/pull/28733/commits/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a)) - ✅

⏲️  Total pipeline duration: 07mn14s 

| Step                                                                       | Result   |
|:---------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-csv/metadata.yaml     | ✅        |
| Connector version semver check                                             | ✅        |
| QA checks                                                                  | ✅        |
| Build connector tar                                                        | ✅        |
| Build destination-csv docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-csv:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690380030/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a/destination-csv/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-csv test
```

File: airbyte-db/db-lib/src/main/java/io/airbyte/db/mongodb/MongoUtils.java [hunk 1]
```diff
@@ -316,15 +319,17 @@ private static List<String> getFieldsName(final MongoCollection<Document> collec
 
   private static List<String> getTypes(final MongoCollection<Document> collection, final String name) {
     final var fieldName = "$" + name;
+    LOGGER.debug("Fetching types for field '{}' (name '{}')...", fieldName, name);
     final AggregateIterable<Document> output = collection.aggregate(Arrays.asList(
         new Document("$limit", DISCOVER_LIMIT),
-        new Document("$project", new Document(ID, 0).append("fieldType", new Document("$type", fieldName))),
+        new Document("$project", new Document(ID, 0).append("fieldType", new Document("$type", cleanFieldName(fieldName)))),
```
[the remark this answers is not in the document]
Comment (jdpgrailsdev) [reply airbyte-db/db-lib/src/main/java/io/airbyte/db/mongodb/MongoUtils.java:325]: @jpefaur I saw that also.  I think there is a subtle difference there -- the other occurrence is creating an `$objectToArray` document and not a `fieldType`, so I suspect its something to do with that.  I tested with and without the change locally and was able to reproduce the issue and then fix it by simply removing the `$` from this one code path.

**Comment by octavia-squidington-iii** [comment]
##  source-tidb test report (commit [`58e1ec7ad3`](https://github.com/airbytehq/airbyte/pull/28733/commits/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a)) - ❌

⏲️  Total pipeline duration: 09mn20s 

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-tidb/metadata.yaml     | ✅        |
| Connector version semver check                                         | ✅        |
| QA checks                                                              | ✅        |
| Build connector tar                                                    | ✅        |
| Build source-tidb docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-tidb:integrationTest | ✅        |
| Acceptance tests                                                       | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690380030/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a/source-tidb/0.2.5/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-tidb test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-r2 test report (commit [`58e1ec7ad3`](https://github.com/airbytehq/airbyte/pull/28733/commits/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a)) - ❌

⏲️  Total pipeline duration: 08mn09s 

| Step                                                                      | Result   |
|:--------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-r2/metadata.yaml     | ✅        |
| Connector version semver check                                            | ✅        |
| QA checks                                                                 | ✅        |
| Build connector tar                                                       | ✅        |
| Build destination-r2 docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-r2:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690380030/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a/destination-r2/0.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-r2 test
```

**Comment by octavia-squidington-iii** [comment]
##  source-alloydb test report (commit [`58e1ec7ad3`](https://github.com/airbytehq/airbyte/pull/28733/commits/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a)) - ❌

⏲️  Total pipeline duration: 08mn15s 

| Step                                                                      | Result   |
|:--------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-alloydb/metadata.yaml     | ✅        |
| Connector version semver check                                            | ✅        |
| QA checks                                                                 | ✅        |
| Build connector tar                                                       | ✅        |
| Build source-alloydb docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-alloydb:integrationTest | ✅        |
| Acceptance tests                                                          | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690380030/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a/source-alloydb/3.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-alloydb test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-redshift test report (commit [`58e1ec7ad3`](https://github.com/airbytehq/airbyte/pull/28733/commits/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a)) - ✅

⏲️  Total pipeline duration: 38mn00s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-redshift/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-redshift docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization-redshift:dev                                        | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-redshift:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690380030/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a/destination-redshift/0.6.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-redshift test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-tidb test report (commit [`58e1ec7ad3`](https://github.com/airbytehq/airbyte/pull/28733/commits/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a)) - ✅

⏲️  Total pipeline duration: 24mn30s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-tidb/metadata.yaml     | ✅        |
| Connector version semver check                                              | ✅        |
| QA checks                                                                   | ✅        |
| Build connector tar                                                         | ✅        |
| Build destination-tidb docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization-tidb:dev                                        | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-tidb:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690380030/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a/destination-tidb/0.1.4/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-tidb test
```

File: airbyte-db/db-lib/src/main/java/io/airbyte/db/mongodb/MongoUtils.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (jdpgrailsdev) [reply airbyte-db/db-lib/src/main/java/io/airbyte/db/mongodb/MongoUtils.java]: @jpefaur PR updated to remove the addition of the `$`.  I tested a discover call locally and it worked.

**Review by jpefaur — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-bigquery test report (commit [`58e1ec7ad3`](https://github.com/airbytehq/airbyte/pull/28733/commits/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a)) - ✅

⏲️  Total pipeline duration: 18mn02s 

| Step                                                                       | Result   |
|:---------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-bigquery/metadata.yaml     | ✅        |
| Connector version semver check                                             | ✅        |
| QA checks                                                                  | ✅        |
| Build connector tar                                                        | ✅        |
| Build source-bigquery docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-bigquery:integrationTest | ✅        |
| Acceptance tests                                                           | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690380030/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a/source-bigquery/0.3.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-bigquery test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-mqtt test report (commit [`58e1ec7ad3`](https://github.com/airbytehq/airbyte/pull/28733/commits/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a)) - ❌

⏲️  Total pipeline duration: 22mn39s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-mqtt/metadata.yaml     | ✅        |
| Connector version semver check                                              | ✅        |
| QA checks                                                                   | ✅        |
| Build connector tar                                                         | ✅        |
| Build destination-mqtt docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-mqtt:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690380030/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a/destination-mqtt/0.1.3/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-mqtt test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-pubsub test report (commit [`58e1ec7ad3`](https://github.com/airbytehq/airbyte/pull/28733/commits/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a)) - ✅

⏲️  Total pipeline duration: 18mn19s 

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-pubsub/metadata.yaml     | ✅        |
| Connector version semver check                                                | ✅        |
| QA checks                                                                     | ✅        |
| Build connector tar                                                           | ✅        |
| Build destination-pubsub docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-pubsub:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690380030/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a/destination-pubsub/0.2.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-pubsub test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-elasticsearch test report (commit [`58e1ec7ad3`](https://github.com/airbytehq/airbyte/pull/28733/commits/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a)) - ❌

⏲️  Total pipeline duration: 09mn41s 

| Step                                                                                 | Result   |
|:-------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-elasticsearch/metadata.yaml     | ✅        |
| Connector version semver check                                                       | ✅        |
| QA checks                                                                            | ✅        |
| Build connector tar                                                                  | ✅        |
| Build destination-elasticsearch docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-elasticsearch:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690380030/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a/destination-elasticsearch/0.1.6/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-elasticsearch test
```

**Comment by octavia-approvington** [comment]
Lets merge it <br/> sheep thumbs up

**Review by octavia-approvington — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-mssql-strict-encrypt test report (commit [`58e1ec7ad3`](https://github.com/airbytehq/airbyte/pull/28733/commits/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a)) - ❌

⏲️  Total pipeline duration: 18mn39s 

| Step                                                                                   | Result   |
|:---------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-mssql-strict-encrypt/metadata.yaml     | ✅        |
| Connector version semver check                                                         | ✅        |
| QA checks                                                                              | ✅        |
| Build connector tar                                                                    | ✅        |
| Build source-mssql-strict-encrypt docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-mssql-strict-encrypt:integrationTest | ❌        |
| Acceptance tests                                                                       | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690380030/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a/source-mssql-strict-encrypt/1.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mssql-strict-encrypt test
```

**Comment by octavia-squidington-iii** [comment]
##  source-postgres-strict-encrypt test report (commit [`58e1ec7ad3`](https://github.com/airbytehq/airbyte/pull/28733/commits/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a)) - ❌

⏲️  Total pipeline duration: 10mn34s 

| Step                                                                                      | Result   |
|:------------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-postgres-strict-encrypt/metadata.yaml     | ✅        |
| Connector version semver check                                                            | ✅        |
| QA checks                                                                                 | ✅        |
| Build connector tar                                                                       | ✅        |
| Build source-postgres-strict-encrypt docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-postgres-strict-encrypt:integrationTest | ✅        |
| Acceptance tests                                                                          | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690380030/58e1ec7ad361d0bb29d42b9d597fc8ab2c92b03a/source-postgres-strict-encrypt/3.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres-strict-encrypt test
```

**Comment by octavia-squidington-iii** [comment]
##  source-e2e-test test report (commit [`fcba3eeb65`](https://github.com/airbytehq/airbyte/pull/28733/commits/fcba3eeb65d56407e4977e165964727f70b2b8e2)) - ✅

⏲️  Total pipeline duration: 07mn12s 

| Step                                                                       | Result   |
|:---------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-e2e-test/metadata.yaml     | ✅        |
| Connector version semver check                                             | ✅        |
| QA checks                                                                  | ✅        |
| Build connector tar                                                        | ✅        |
| Build source-e2e-test docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-e2e-test:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690402518/fcba3eeb65d56407e4977e165964727f70b2b8e2/source-e2e-test/2.1.4/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-e2e-test test
```

**Comment by octavia-squidington-iii** [comment]
##  source-oracle test report (commit [`fcba3eeb65`](https://github.com/airbytehq/airbyte/pull/28733/commits/fcba3eeb65d56407e4977e165964727f70b2b8e2)) - ❌

⏲️  Total pipeline duration: 08mn39s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-oracle/metadata.yaml     | ✅        |
| Connector version semver check                                           | ✅        |
| QA checks                                                                | ✅        |
| Build connector tar                                                      | ✅        |
| Build source-oracle docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-oracle:integrationTest | ❌        |
| Acceptance tests                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690402518/fcba3eeb65d56407e4977e165964727f70b2b8e2/source-oracle/0.4.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-oracle test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-selectdb test report (commit [`fcba3eeb65`](https://github.com/airbytehq/airbyte/pull/28733/commits/fcba3eeb65d56407e4977e165964727f70b2b8e2)) - ❌

⏲️  Total pipeline duration: 22mn58s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-selectdb/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| QA checks                                                                       | ❌        |
| Build connector tar                                                             | ✅        |
| Build destination-selectdb docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-selectdb:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690402518/fcba3eeb65d56407e4977e165964727f70b2b8e2/destination-selectdb/0.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-selectdb test
```

**Comment by octavia-squidington-iii** [comment]
##  source-redshift test report (commit [`fcba3eeb65`](https://github.com/airbytehq/airbyte/pull/28733/commits/fcba3eeb65d56407e4977e165964727f70b2b8e2)) - ❌

⏲️  Total pipeline duration: 17mn14s 

| Step                                                                       | Result   |
|:---------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-redshift/metadata.yaml     | ✅        |
| Connector version semver check                                             | ✅        |
| QA checks                                                                  | ✅        |
| Build connector tar                                                        | ✅        |
| Build source-redshift docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-redshift:integrationTest | ❌        |
| Acceptance tests                                                           | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690402518/fcba3eeb65d56407e4977e165964727f70b2b8e2/source-redshift/0.4.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-redshift test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-redshift test report (commit [`fcba3eeb65`](https://github.com/airbytehq/airbyte/pull/28733/commits/fcba3eeb65d56407e4977e165964727f70b2b8e2)) - ✅

⏲️  Total pipeline duration: 46mn36s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-redshift/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-redshift docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization-redshift:dev                                        | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-redshift:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690402518/fcba3eeb65d56407e4977e165964727f70b2b8e2/destination-redshift/0.6.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-redshift test
```

**Comment by octavia-squidington-iii** [comment]
##  source-postgres test report (commit [`fcba3eeb65`](https://github.com/airbytehq/airbyte/pull/28733/commits/fcba3eeb65d56407e4977e165964727f70b2b8e2)) - ✅

⏲️  Total pipeline duration: 20mn01s 

| Step                                                                       | Result   |
|:---------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-postgres/metadata.yaml     | ✅        |
| Connector version semver check                                             | ✅        |
| QA checks                                                                  | ✅        |
| Build connector tar                                                        | ✅        |
| Build source-postgres docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-postgres:integrationTest | ✅        |
| Acceptance tests                                                           | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690402518/fcba3eeb65d56407e4977e165964727f70b2b8e2/source-postgres/3.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres test
```

**Comment by octavia-squidington-iii** [comment]
##  source-e2e-test-cloud test report (commit [`fcba3eeb65`](https://github.com/airbytehq/airbyte/pull/28733/commits/fcba3eeb65d56407e4977e165964727f70b2b8e2)) - ❌

⏲️  Total pipeline duration: 06mn23s 

| Step                                                                             | Result   |
|:---------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-e2e-test-cloud/metadata.yaml     | ✅        |
| Connector version semver check                                                   | ✅        |
| QA checks                                                                        | ❌        |
| Build connector tar                                                              | ✅        |
| Build source-e2e-test-cloud docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-e2e-test-cloud:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690402518/fcba3eeb65d56407e4977e165964727f70b2b8e2/source-e2e-test-cloud/2.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-e2e-test-cloud test
```

**Comment by octavia-squidington-iii** [comment]
##  source-db2 test report (commit [`fcba3eeb65`](https://github.com/airbytehq/airbyte/pull/28733/commits/fcba3eeb65d56407e4977e165964727f70b2b8e2)) - ❌

⏲️  Total pipeline duration: 12mn16s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-db2/metadata.yaml     | ✅        |
| Connector version semver check                                        | ✅        |
| QA checks                                                             | ✅        |
| Build connector tar                                                   | ✅        |
| Build source-db2 docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-db2:integrationTest | ❌        |
| Acceptance tests                                                      | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_handle-invalid-field-paths/1690402518/fcba3eeb65d56407e4977e165964727f70b2b8e2/source-db2/0.1.20/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-db2 test
```

**Comment by hash-data** [comment]
hi @jdpgrailsdev  how you run it locally without building docker image

**Pull request merged**